### PR TITLE
Add a new component Headline

### PR DESCRIPTION
**Pull request by mysterious25**
Hello, check my code please.

Thank you)

### CODE REVIEW

File: src/components/common/Button/Button.module.scss [hunk 1]
```diff
@@ -1,10 +1,4 @@
 
-
```
Comment (savtym) [inline src/components/common/Button/Button.module.scss:2]: Why did you remove these classes?

File: src/components/common/Headline/Headline.js [hunk 2]
```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+
+function Headline() {
+    return {
+        <h1 type="post_title"/>
```
Comment (savtym) [inline src/components/common/Headline/Headline.js]: what is an attribute `type`?

File: src/components/common/Headline/Headline.js [hunk 3]
```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+
+function Headline() {
```
Comment (savtym) [inline src/components/common/Headline/Headline.js]: What does this component do?
Just render `h1` tag?

File: src/components/common/Headline/Headline.module.scss [hunk 4]
```diff
@@ -0,0 +1,10 @@
+
```
Comment (savtym) [inline src/components/common/Headline/Headline.module.scss]: What is it?

File: src/components/core/Main/Main.js [hunk 5]
```diff
@@ -5,13 +5,16 @@ import {
...
 	TypesButton,
+	Headline,
 } from 'components/common';
 
 
 class Main extends Component {
 	render() {
 		return (
 			<div className={Styles.wrapper}>
+				<h1 className={Styles.headline}>I added new component h1 </h1>
```
Comment (savtym) [inline src/components/core/Main/Main.js]: And you import `Headline` component, but you don't use it.

File: src/components/core/Main/Main.module.scss [hunk 6]
```diff
@@ -8,3 +8,7 @@
 .title {
   color: $gray;
 }
+
+.headline {
+  color: green;
```
Comment (savtym) [inline src/components/core/Main/Main.module.scss:13]: I recommend using a variable with colors.
You can add a new variable in file `colors.scss` and using it.

File: src/components/common/Headline/Headline.js (hunk 3 above)
Comment (mysterious25) [reply src/components/common/Headline/Headline.js]: yes

Comment (savtym) [reply src/components/common/Headline/Headline.js]: Where you set some text?

File: src/components/common/Button/Button.module.scss (hunk 1 above)
Comment (mysterious25) [reply src/components/common/Button/Button.module.scss:2]: this is wrong, now i fixed it

File: src/components/common/Headline/Headline.js (hunk 2 above)
Comment (mysterious25) [reply src/components/common/Headline/Headline.js]: now I wrote this code className={post_title}

Comment (savtym) [reply src/components/common/Headline/Headline.js]: I try to understand you
Do you want to add className `post_title`?

Comment (mysterious25) [reply src/components/common/Headline/Headline.js]: yes

File: src/components/common/Headline/Headline.js [hunk 7]
```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+
+function Headline(props) {
+    const name = 'hi its variable things';
+    const { children, type } = props;
+
+    return (
+        <h1> {children} Компонента {name} </h1>
```
Comment (savtym) [inline src/components/common/Headline/Headline.js]: just {children} without another text

File: src/components/common/Headline/Headline.js [hunk 8]
```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+
```
Comment (savtym) [inline src/components/common/Headline/Headline.js:3]: add styles for Headline

File: src/components/core/Main/Main.js [hunk 9]
```diff
@@ -4,24 +4,32 @@ import Styles from './Main.module.scss';
...
+import Headline from 'components/common/Headline/Headline';
 
 class Main extends Component {
 	render() {
 		return (
 			<div className={Styles.wrapper}>
+
 				<span>Hello World!</span>
 
+                <Headline name="tralalalal" />
```
Comment (savtym) [inline src/components/core/Main/Main.js]: remove this

File: src/components/core/Main/Main.js [hunk 10]
```diff
@@ -4,24 +4,32 @@ import Styles from './Main.module.scss';
...
+
 				<Button type={TypesButton.error}>
 					<TextField/>
 				</Button>
 
 				<Button type={TypesButton.success}>
 					<span className={Styles.error}>Error</span>
 				</Button>
-
+                <Headline >fffffff
```
Comment (savtym) [inline src/components/core/Main/Main.js]: refactor to readable code

File: src/components/core/Main/Main.js [hunk 11]
```diff
@@ -4,24 +4,32 @@ import Styles from './Main.module.scss';
 import {
 	Button,
 	TextField,
-	TypesButton,
+	TypesButton
 } from 'components/common';
 
+import Headline from 'components/common/Headline/Headline';
```
Comment (savtym) [inline src/components/core/Main/Main.js:10]: you need to add export in `src/components/common/index.js`
after, you export from `components/common`

File: src/components/common/Headline/Headline.module.scss (hunk 4 above)
Comment (mysterious25) [reply src/components/common/Headline/Headline.module.scss]: i fixed it

File: src/components/common/Headline/Headline.js [hunk 12]
```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import Styles from './Headline.module.scss';
```
Comment (savtym) [inline src/components/common/Headline/Headline.js:2]: add a new empty line between `import React` and `import Styles`

File: src/components/common/Headline/Headline.module.scss [hunk 13]
```diff
@@ -0,0 +1,11 @@
...
+  color: #F44336;
+}
+
+.headline {
+  color: #004D40;
+}
+
+.green{
+  color: #004D40;
+}
```
Comment (savtym) [inline src/components/common/Headline/Headline.module.scss:11]: Why do you add `.red` and `.green` classes?

File: src/components/core/Main/Main.js [hunk 14]
```diff
@@ -4,24 +4,32 @@ import Styles from './Main.module.scss';
...
 
 class Main extends Component {
 	render() {
 		return (
 			<div className={Styles.wrapper}>
+
 				<span>Hello World!</span>
 
+                <Headline/>
```
Comment (savtym) [inline src/components/core/Main/Main.js:19]: remove this line

**Comment by mysterious25** [comment]
Thank you for comments
